project/tests/test_first_employees_working_hours/test_first_employees_working_hours.py

```diff
@@ -39,7 +39,7 @@ def test_first_employees_working_hours():
         meta = json.load(f)
 
     res = requests.post(os.getenv('urlpost'), files=files)
-    time.sleep(20)
+    time.sleep(30)
```

Comment:
do we need to wair here?  isn't it a sync request?

Reply:
yes, you need to wait for the api to calculate the result

Reply:
ok